creme/settings.py

```diff
@@ -8,7 +8,6 @@
 from django.utils.translation import gettext_lazy as _
 
 DEBUG = False
-JAVASCRIPT_DEBUG = DEBUG
```

Comment:
I'd prefer a separated patch with an entry in the Changelog ("unused setting XXX was removed ..."